docs/src/examples/components/Ref/Types/RefExampleForwardRef.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Grid, Ref, Segment } from '@stardust-ui/react'
+
+const ExampleButton = React.forwardRef<HTMLButtonElement>((props, ref) => (
+  <div>
+    <button {...props} ref={ref} />
+  </div>
+))
+
+class RefExampleForwardRef extends React.Component {
```

Comment:
actually, very confused by trying to understand how `Ref` component is helpful in case of ref forwarding (and this, as stated in the description, is its main purpose: `This PR is redesign of Ref component to use forwardRef API by default and use findDOMNode() only as a fallback.`)

Let me explain resaons for this by following the example. Suppose that I am a Stardust client and have the following component: 

```jsx
const MyButton = () => (
  <div>
    <button {...props} ref={ref} />
  </div>
)
```

With this component I already expect the following to be true:

```jsx
// when mounted, this.ref.current will refer to <button />
<MyButton ref={this.ref} />
```

Then comes Stardust and proposes to use `Ref` - with no clear benefits to me, as now it will be either:
- I will use `Ref` as **_another_** wrapper - but what are the benefits of that :/ ?
```jsx
const WithRef = () => (
  <Ref>
    { React.forwardRef((props, ref) => <MyButton ref={ref} />
  </Ref>
)
```
- or what is the scenario where I might need it? 

----------------

**`Ref`, in case of ref forwarding case, acts as a simple `ref` translator** - so, what are the reasons I wouldn't use built-in React ref forwarding capabilities and would prefer `Ref` component abstraction instead?